app/src/main/java/io/gnosis/safe/ui/settings/owner/list/OwnerListViewModel.kt

```diff
@@ -70,7 +70,7 @@ class OwnerListViewModel
                         .sortedBy { it.name }
                 val acceptedOwners = owners.filter { localOwner ->
                     safe.signingOwners.any {
-                        localOwner.address == it
+                        localOwner.address == it && localOwner.type != Owner.Type.LEDGER_NANO_X
```

Comment:
here adding FIXME or TODO comment would make sense. Something like: Allow ledger keys to be preselected after ledger execution flow is adjusted.